src/modules/Dashboard/features/RespondentData/RespondentDataSummary/hooks/useRespondentAnswers/useRespondentAnswers.ts

```diff
@@ -21,62 +21,92 @@ export const useRespondentAnswers = () => {
     startTime: providedStartTime,
     endTime: providedEndTime,
     filterByIdentifier: providedFilterByIdentifier,
+    identifier: providedIdentifier,
     versions: providedVersions,
+    isIdentifiersChange = false,
   }: FetchAnswers) => {
     if (!appletId || !respondentId) return;
+
     try {
       const {
-        startDate,
+        startDate: formStartDate,
         endDate: formEndDate,
         startTime: formStartTime,
         endTime: formEndTime,
-        identifier,
+        identifier: formIdentifier,
         filterByIdentifier: formFilterByIdentifier,
         versions: formVersions,
         identifiers,
       } = getValues();
-      const endDate = providedEndDate ?? formEndDate;
       const startTime = providedStartTime ?? formStartTime;
       const endTime = providedEndTime ?? formEndTime;
       const filterByIdentifier = providedFilterByIdentifier ?? formFilterByIdentifier;
       const versions = providedVersions ?? formVersions;
+      const identifier = providedIdentifier ?? formIdentifier;
```

Comment:
Could you please make sure that test is cover the case when `providedIdentifier` is undefined/null?